apps/api/src/users/users.controller.ts

```diff
@@ -36,4 +52,74 @@ export class UsersController {
   ): Promise<AccessTokenResponse> {
     return this.usersService.updateUserPassword(user, updateUserPasswordDto);
   }
+
+  @UseGuards(JwtAuthGuard)
```

Comment:
Protecting this route makes it hard to fetch the profile picture, because it would require to use `fetch()` with the authorization header set.